docs/getting-started/zowe_v2_faq.md

```diff
@@ -6,4 +6,30 @@ The official date is TBD, the target is April 25, 2022; look for the official an
 
 ## Where can I find the current (V1) and new (V2) LTS conformancee criteria? 
 
-The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
\ No newline at end of file
+The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
```

Comment:
You should reword "Use the light-GREEN highlights to easily identify the changes." 
to "Use the included fill color key to identify new changes for V2, reworded changes, or changes from V1 removed in V2" because we now have green, yellow, and red in color key